1.23.5 does not compile with `aead-cipher` feature disabled

I'm trying to build with such flags:
```
$ cargo build --release --bin ssservice --no-default-features --features 'service,logging,multi-threaded,aead-cipher-2022'
   Compiling shadowsocks-service v1.23.5 (/tmp/shadowsocks-rust/crates/shadowsocks-service)
error[E0308]: mismatched types
   --> crates/shadowsocks-service/src/manager/server.rs:482:43
    |
482 |             None => return Ok(AddResponse("method is required")),
    |                               ----------- ^^^^^^^^^^^^^^^^^^^^- help: try using a conversion method: `.to_string()`
    |                               |           |
    |                               |           expected `String`, found `&str`
    |                               arguments to this struct are incorrect
    |
note: tuple struct defined here
   --> /tmp/shadowsocks-rust/crates/shadowsocks/src/manager/protocol.rs:74:12
    |
74  | pub struct AddResponse(pub String);
    |            ^^^^^^^^^^^

For more information about this error, try `rustc --explain E0308`.
error: could not compile `shadowsocks-service` (lib) due to 1 previous error
```
Rust compiler is up to date:
```
$ rustc --version
rustc 1.88.0 (6b00bc388 2025-06-23) (Arch Linux rust 1:1.88.0-1)
```
Previous versions compiled just fine.

--- a/crates/shadowsocks-service/src/manager/server.rs
+++ b/crates/shadowsocks-service/src/manager/server.rs
@@ -479,7 +479,7 @@ impl Manager {
             #[cfg(feature = "aead-cipher")]
             None => self.svr_cfg.method.unwrap_or(CipherKind::CHACHA20_POLY1305),
             #[cfg(not(feature = "aead-cipher"))]
-            None => return Ok(AddResponse("method is required")),
+            None => return Ok(AddResponse("method is required".to_string())),
         };
 
         let mut svr_cfg = match ServerConfig::new(addr, req.password.clone(), method) {

